examples/fox_in_a_hole/fox_in_a_hole.py

```diff
@@ -191,58 +283,44 @@ def check_guess(self, guess) -> bool:
     def take_random_move(self) -> str:
         """Applies a random move on the current state. Gives back the move in string format."""
         probs = self.state[0].get_binary_probabilities(objects=self.state[1])
-        non_empty_holes = []
-        for i, prob in enumerate(probs):
-            if prob > 0:
-                non_empty_holes.append(i)
+        non_empty_holes = [i for i in range(len(probs)) if probs[i] > 0]
```

Comment:
list comprehension can be made nicer with `[i for i, p in enumerate(probs) if p > 0]`

Reply:
Done.